### DISCUSSION TITLE
OpenIM community is recruiting new members! Join us to contribute to open source projects and improve your engineering skills! 🎉

### DISCUSSION THREAD

**Discussion by cubxxw** [openimsdk/open-im-server#426] [Show and tell]
## Join the OpenIM community 🎉

👋 We welcome more developers and students to join the OpenIM community, contribute to open source projects, learn new technologies, and improve engineering capabilities. If you want to join us, you can do so in the following ways:

### Join the OpenIM project 🤝

💡 You can contribute to any project or feature module of OpenIM. Collaborative development can help improve your technical skills and engineering capabilities, and add rich experience to your resume.

### Create a new repository 🚀

📚 If you have your own project that you want to make public, you can use our scaffolding and templates to create a new repository and make it public. This can help you get a better development experience, and find more developers and friends with common goals and aspirations.

### Join Slack 💻

👨‍💻 You can join our [Slack](https://join.slack.com/t/openimsdk/shared_invite/zt-22720d66b-o_FvKxMTGXtcnnnHiMqe9Q) to ask questions and get help from our community, as well as discuss and share your ideas and suggestions.

### How to join? 🤖

👩‍💻 If you want to join us, please comment below and use the following template:

```
+ 🤵 English name:
+ 🧶 GitHub username:
+ 📢 Slack username:
+ 📮 Email address:
+ 🎯 Skill Points:
+ 🏙️ City:
```

🎉 We will contact you as soon as possible. Welcome to the OpenIM community!🎊

🙌🌟👏🔥

**Comment by money8203** [comment]
Awesome, would love to be a part of the community!!

Name: Manish Tyagi
GitHub username: money8203
Email address: manishtyagi088@gmail.com
City: Greater Noida, India

**Reply by cubxxw** [reply]
hi @money8203 ,Would you please join us on slack and talk about it
https://join.slack.com/t/openimsdk/shared_invite/zt-1tmoj26uf-_FDy3dowVHBiGvLk9e5Xkg

**Comment by ridewind88** [comment]
Name: Glory
GitHub Username: distancemountain
Email Address: [glory9968@163.com](mailto:glory9968@163.com)
City: Shenzhen

**Reply by cubxxw** [reply]
hi @distancemountain,Which job would you like to join? Would you please join us on slack and talk about it
[join.slack.com/t/openimsdk/shared_invite/zt-1tmoj26uf-_FDy3dowVHBiGvLk9e5Xkg](https://join.slack.com/t/openimsdk/shared_invite/zt-1tmoj26uf-_FDy3dowVHBiGvLk9e5Xkg)

**Comment by IRONICBo** [comment]
I am very proud to be a part of the OpenIM community! :)

Name: Asklv
Github username: IRONICBo
email address: bbemailspider@163.com
City: Wuhan, China

**Reply by cubxxw** [reply]
slack: Asklv

**Comment by sjcsjc123** [comment]
Name：loubens
Github username: sjcsjc123
email address: loubens@163.com
City: Hangzhou, China

**Comment by PolarishT** [comment]
name: AllianceTing
githubName: AllianceTing
emailAddress: 2426446427@qq.com
city:tianjin,China

**Comment by qishenonly** [comment]
Name：GodQI
Github username: qishenonly
email address: bsgodqi@163.com
City: HeFei , China

**Comment by ZhFngHui** [comment]
Name：F.H. Zhang
Github username: ZhFngHui
email address: [zfh19980305@163.com] (mailto:19980305@163.com)
City: ShenZhen , China

**Comment by ScottLinnn** [comment]
+ 🤵 English Name: Shuning Lin
+ 🧶 GitHub Username: ScottLinnn
+ 📢 Slack Username: Shuning Lin
+ 📮 Email Address: shuninglin5@gmail.com
+ 🎯 Skill Points: C++
+ 🏙️ City: Beijing, China

**Comment by Daz-3ux** [comment]
+ 🤵 English name: Daz
+ 🧶 GitHub username: Daz-3ux
+ 📢 Slack username: Daz
+ 📮 Email address: daz-3ux@proton.me
+ 🎯 Skill Points: Go, C, Python, Docker/K8s
+ 🏙️ City: Xian, Shaanxi

**Comment by PapayaHUANG** [comment]
+ 🤵 English name: Papaya
+ 🧶 GitHub username: PapayaHUANG
+ 📢 Slack username: N/A
+ 📮 Email address: shaoyahuang22@gmail.com
+ 🎯 Skill Points: JavaScript, TypeScript, React
+ 🏙️ City: Shanghai

**Comment by tobehardest** [comment]
+ 🤵 English name: Yikun Chen
+ 🧶 GitHub username: Yikun Chen
+ 📢 Slack username: Yikun Chen
+ 📮 Email address: cvchen587@163.com
+ 🎯 Skill Points: Java, golang
+ 🏙️ City: Guangzhou，ShenZhen

**Comment by eurecalulu** [comment]
I am very proud to be a part of the OpenIM community! :)

Name: Lu Cao
Github username: eurecalulu
email address: eurecalulu@gmail.com
City: Nanjing, China

**Comment by StellarisW** [comment]
🤵 English name: Charles Wang
🧶 GitHub username: Stellarisw
📢 Slack username: Stellarisw
📮 Email address: a@stellaris.wang
🎯 Skill Points: golang
🏙️ City: Chongqing, China

**Comment by plutoyty** [comment]
+ 🧶 GitHub username: plutoyty
+ 📢 Slack username: plutoyty
+ 📮 Email address: plutoyty@163.com
+ 🎯 Skill Points: Golang,Java
+ 🏙️ City: ShenZhen , China

**Comment by xionyz** [comment]
🤵 English name: Euan
🧶 GitHub username: xionyz
📮 Email address: 1895650697@qq.com
🎯 Skill Points: Go,PHP
🏙️ City: ChengDu, China

**Comment by wennjie** [comment]
🤵 English Name: Jojo
🧶 GitHub Username: wennjie
📢 Slack Username: [wenjieskylin@gmail.com](mailto:wenjieskylin@gmail.com)
📮 Email Address: [wenjieskylin@gmail.com](mailto:wenjieskylin@gmail.com)
🎯 Skills: JavaScript, TypeScript, React, Vue, ReactNative
🏙️ City: Chengdu, China

**Reply by cubxxw** [reply]
Thank you for your comments, and welcome to read our contributor documentation to learn about OpenIM, Join the slack https://join.slack.com/t/openimsdk/shared_invite/zt-22720d66b-o_FvKxMTGXtcnnnHiMqe9Q developers

**Comment by ysxiang12** [comment]
🤵 English Name: Ah Xiang
🧶 GitHub Username: ysxiang12
📢 Slack Username: [Not provided]
📮 Email Address: [1076656527@qq.com](mailto:1076656527@qq.com)
🎯 Skills: JavaScript, TypeScript, React, Vue, Mini Program
🏙️ City: Chongqing, China

**Reply by cubxxw** [reply]
Thank you for your comments, and welcome to read our contributor documentation to learn about OpenIM, Join the slack https://join.slack.com/t/openimsdk/shared_invite/zt-22720d66b-o_FvKxMTGXtcnnnHiMqe9Q developers
or my wechat: nsddd_top

**Comment by zengxilong** [comment]
+ 🤵 English name: Jasper Zeng
+ 🧶 GitHub username: zengxilong
+ 📢 Slack username: Jasper Zeng
+ 📮 Email address: zengxilonglh@gmail.com
+ 🎯 Skill Points: Golang
+ 🏙️ City: Shenzhen, China

**Comment by skiffer-git** [comment]
🤵 English name: skiffer
🧶 GitHub username: skiffer-git
📢 Slack username: OpenIM-skiffer
📮 Email address: winxu81@gmail.com
🎯 Skill Points: Golang
🏙️ City: Shenzhen, China

**Comment by std-s** [comment]
+ 🤵 English name: Brett
+ 🧶 GitHub username: std-s
+ 📢 Slack username: OpenIM-Brett
+ 📮 Email address: [e001.player@gmail.com](mailto:e001.player@gmail.com)
+ 🎯 Skill Points: iOS，Flutter
+ 🏙️ City: Shenzhen, China

**Comment by DreamerXXT** [comment]
+ 🤵 English name: xiaoting
+ 🧶 GitHub username: DreamerXXT
+ 📢 Slack username: OpenIM-xt
+ 📮 Email address: x13686931575@gmail.com
+ 🎯 Skill Points: pytest
+ 🏙️ City:Shenzhen,China

**Comment by withchao** [comment]
🤵 English name: chao
🧶 GitHub username: withchao
📢 Slack username: chao
📮 Email address: 993506633@qq.com
🎯 Skill Points: Golang
🏙️ City: Shenzhen, China

**Comment by Bloomingg** [comment]
🤵 English name: blooming
🧶 GitHub username: Bloomingg
📢 Slack username: OpenIM-blooming
📮 Email address: [blooming2477@gmail.com](mailto:blooming2477@gmail.com)
🎯 Skill Points: Web
🏙️ City: Shenzhen, China

**Comment by lgz5689** [comment]
🤵 English name: kevin
🧶 GitHub username: lgz5689
📢 Slack username: OpenIM-Kevin
📮 Email address: leegz5689@gmail.com
🎯 Skill Points: Web
🏙️ City: Shenzhen, China

**Comment by CNCSMonster** [comment]
English name: CCM
Github username: cncsmonster
Slack username: cncsmonster
Email address: cncsmonster@163.com
Skill Points: Web,Compiler
City: Shenzhen, China

**Comment by luhaoling** [comment]
🤵 English name: Brabem
🧶 GitHub username: luhaoling
📢 Slack username: Brabem
📮 Email address: 2198702716@qq.com
🎯 Skill Points: Golang
🏙️ City: Shenzhen, China

**Comment by FGadvancer** [comment]
+ 🤵 English name: Gordon
+ 🧶 GitHub username: FGadvancer
+ 📢 Slack username: Gordon
+ 📮 Email address:[a1432970085@gmail.com](mailto:a1432970085@gmail.com)
+ 🎯 Skill Points: Golang
+ 🏙️ City: Shenzhen, China

**Comment by Oliver-WJ** [comment]
+ 🤵 English name:Oliver
+ 🧶 GitHub username:Oliver Wong
+ 📢 Slack username:Oliver
+ 📮 Email address:WANGJIN_1163@163.com
+ 🎯 Skill Points:Java、Kotlin、dart
+ 🏙️ City:Chengdu,China

**Comment by luhaoling** [comment]
🤵 English name: Brabem
🧶 GitHub username: luhaoling
📢 Slack username: Brabem
📮 Email address: lmf91248@gmail.com
🎯 Skill Points: Golang
🏙️ City: Shenzhen, China

**Comment by Xx7x1** [comment]
🤵 English name: Vincy
🧶 GitHub username: Xx7x1
📢 Slack username: Xxin
📮 Email address: xxin30063@gmail.com
🎯 Skill Points: UI
🏙️ City: Shenzhen, China

**Comment by xuexihuang** [comment]
🤵 English name: tim
🧶 GitHub username: xuexihuang
📢 Slack username: tim
📮 Email address: g1339326187@gmail.com
🎯 Skill Points: golang,k8s
🏙️ City: Shenzhen, China

**Comment by sixhj** [comment]
Name: sixhj
GitHub Username: sixhj
Email Address: sixhjmain@gmail.com
City: ZhengZhou,China

**Comment by liu-xuewen** [comment]
Name: liuxuewen
GitHub Username: liu-xuewen
Email Address: [liiuxuewen@gmail.com](liiuxuewen@gmail.com)
City: Shenzhen

**Comment by lambiengcode** [comment]
🤵 English name: Kai Dao
🧶 GitHub username: lambiengcode
📢 Slack username: lambiengcode
📮 Email address: lambiengcode@gmail.com
🎯 Skill Points: Flutter, Swift, Kotlin, NodeJS, WebRTC
🏙️ City: HoChiMinh, VietNam

**Reply by cubxxw** [reply]
We are delighted to welcome you to our community! You can join us on Slack at the following link: https://join.slack.com/t/openimsdk/shared_invite/zt-22720d66b-o_FvKxMTGXtcnnnHiMqe9Q

We are always grateful for any contributions you may have to offer. Your contributions, no matter how big or small, are highly valued and appreciated.

Thank you for considering joining us, and we look forward to seeing you soon!

Best regards,
@cubxxw

**Comment by cheungshaopeng** [comment]
🤵 English name: cheungshaopeng
🧶 GitHub username: cheungshaopeng
📢 Slack username: cheungshaopeng
📮 Email address: cheungshaopeng@gmail.com
🎯 Skill Points: c++,golang,python
🏙️ City: Shenzhen, China

**Reply by cubxxw** [reply]
We are delighted to welcome you to our community! You can join us on Slack at the following link: https://join.slack.com/t/openimsdk/shared_invite/zt-22720d66b-o_FvKxMTGXtcnnnHiMqe9Q

join channel: `openim-dev`

**Comment by Lansongxx** [comment]
🤵 English name: Lansong
🧶 GitHub username: Lansongxx
📢 Slack username: Lansong
📮 Email address: [TinnNasseri623@gmail.com](mailto:TinnNasseri623@gmail.com)
🎯 Skill Points: golang
🏙️ City: Guilin, China

**Reply by cubxxw** [reply]
Friends in China can join our openim, support all platforms, you can use the web to enter openim and communicate with us, thank you! 
+ https://web-enterprise.rentsoft.cn/

**Comment by EZ4Jam1n** [comment]
🤵 英文名: benjamin
🧶 GitHub 用户名: EZ4Jam1n
📢 Slack 用户名: benjamin
📮 邮箱: pkubzm@163.com
🎯 技能点: Golang、C++、Java、Vue
🏙️ 城市: 深圳

**Comment by bhlin790** [comment]
+ 🤵 English name: Lynne
+ 🧶 GitHub username: bhlin790
+ 📮 Email address: bhlin790@gamil.com
+ 🎯 Skill Points: Go、Flutter
+ 🏙️ City: NanJing, China
